testsupport/wait/member.go

```diff
@@ -915,7 +915,8 @@ func (a *MemberAwaitility) WaitForPods(namespace string, n int, criteria ...PodW
 			return false, err
 		}
 	pods:
-		for _, p := range foundPods.Items {
+		for i := range foundPods.Items {
+			p := foundPods.Items[i] // avoids the 'G601: Implicit memory aliasing in for loop. (gosec)' problem
 			if !matchPodWaitCriterion(&p, criteria...) {
```

Comment:
the same question - there shouldn't be a problem with the original version, should it?

Reply:
reverted in bc57e8a, using a `nolint:gosec` commend instead